apps/web/pages/auth/login.tsx

```diff
@@ -229,7 +229,7 @@ inferSSRProps<typeof _getServerSideProps> & WithNonceProps<{}>) {
                 type="submit"
                 color="primary"
                 disabled={formState.isSubmitting}
-                className="w-full justify-center">
+                className="w-full justify-center bg-white/70">
```

Comment:
https://github.com/calcom/cal.com/assets/53316345/de6d7d54-72c7-41f9-9245-9ee6815be1b5

This doesn't fix the issue